ingredient/ingredient-kubernetes/src/baketest.yaml

```diff
@@ -0,0 +1,13 @@
+---

```

Comment:
move this into a test folder

Reply:
I've removed it completely. Will address later when I've sorted out my testing strategy a bit better.